packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -151,7 +158,10 @@ export default {
          */
         cardsReceived (cards) {
             if (cards || this.contentCardsHaveLoaded) {
-                this.logInfo(`Content cards loaded: ${cards}`, this.getLogPayload(true));
+                this.$logger.logInfo('f-offers (Results) - Content cards received', this.$store, {
+                    ...this.loggingData,
+                    Count: cards
```

Comment:
Should this have a lowercase "c" at the start?

Reply:
weirdly no lol, this is what the logging system uses as its index...

Reply:
💩 OK, fair enough 😄